datatypes/benches/masked_grid_mapping.rs

```diff
@@ -39,7 +39,7 @@ fn map_indexed_elements_1d_simple(c: &mut Criterion) {
 
 fn map_indexed_elements_1d(c: &mut Criterion) {
     let cpu_cores = std::thread::available_parallelism().unwrap();
-    let thread_nums = (0..=cpu_cores.log2()).map(|exp| 2_usize.pow(exp)); //[1, 2, 4, 8, 16, 32];
+    let thread_nums = (0..=cpu_cores.ilog2()).map(|exp| 2_usize.pow(exp)); //[1, 2, 4, 8, 16, 32];
```

Comment:
What is `ilog2`?

Reply:
integer logarithm. They are changing this on all integer types to make it apparent that something different is done here with respect to rounding errors than on floats.